tests/cpp/test_gpu_view.cpp

```diff
@@ -2584,39 +2584,77 @@ INSTANTIATE_TEST_SUITE_P(
         std::vector<int64_t>{0, 2, 4},
         std::vector<int64_t>{1, 3}));
 
-TEST_F(GpuViewTest, GroupNorm) {
+// GroupNorm with the last reshape moved to output tensors
+// first 3 reshapes are fused in InnerPersistent
+// last reshape is NoOp
+TEST_F(GpuViewTest, GroupNormReshapeMovedToOutput) {
   auto fusion = std::make_unique<Fusion>();
   FusionGuard fg(fusion.get());
   const int64_t N = 2, C = 128, H = 16, W = 16, G = 32;
```

Comment:
Style nit: https://google.github.io/styleguide/cppguide.html#Variable_Names

```suggestion
  constexpr int64_t n = 2, c = 128, h = 16, w = 16, g = 32;
```